CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Changed
 
 - Update Google IMA SDK dependencies to `3.23.0` for iOS, and to `4.13.0` for tvOS
+- Update sample dependency: Google Cast iOS sender SDK to `4.8.1`
```

Comment:
```suggestion
- Update example app dependency: Google Cast iOS sender SDK to `4.8.1`
```

Reply:
Thank you! I was struggling to phrase this 😅